src/main/scala/mesosphere/marathon/state/Group.scala

```diff
@@ -21,7 +21,10 @@ class Group(
     val pods: Map[PathId, PodDefinition] = defaultPods,
     val groupsById: Map[Group.GroupKey, Group] = defaultGroups,
     val dependencies: Set[PathId] = defaultDependencies,
-    val version: Timestamp = defaultVersion) extends IGroup {
+    val version: Timestamp = defaultVersion,
+    val enforceRole: Option[Boolean] = None) extends IGroup {
```

Comment:
I'm not so sure we want it this way. I'd much rather define a top-level group type but I don't know about the use cases so I left it like this.

Reply:
I think it's fine this way. Not sure it would be much cleaner with a separate top-level-group